api/types/crds/skupper_link_access_crd.yaml

```diff
@@ -24,6 +24,9 @@ spec:
                         type: string
                       port:
                         type: integer
+                host:
+                  description: Hostname or IP address that can be used by the chosen access type
```

Comment:
On kubernetes this is supposed to represent the ingress-host, on non-k8s platforms, this can be used as the host name or IP that will be used on respective tokens.
Any suggestion for better naming (or even placing)?

Reply:
I don't think I like combining those two settings in the same field. I think the tweaking of the 'ingress' options will be ingress type dependent and should probably be in options.

Reply:
Options is also something we could use.
One question, though, is that `options` in LinkAccess is set differently than it is on `Site.spec`.
Which one are you in favor of using as a standard?

Reply:
I think an explicit host field for indicating the interface to bind on is fine, even if that is likely to be unused on kubernetes. The thing I don't like is making it a sort of multi-purpose field.

I think the LinkAccess approach of using a map rather than a list of name-value pairs is slightly simpler to compose. The list of name-value pairs does allow multiple entries with the same name however which can be convenient as a way of supplying multi-valued options. I agree we should be consistent. I probably lean to the simple map...

Reply:
> I think an explicit host field for indicating the interface to bind on is fine, even if that is likely to be unused on kubernetes. The thing I don't like is making it a sort of multi-purpose field.

Would it be better to delegate this to the options field then?

> I think the LinkAccess approach of using a map rather than a list of name-value pairs is slightly simpler to compose. The list of name-value pairs does allow multiple entries with the same name however which can be convenient as a way of supplying multi-valued options. I agree we should be consistent. I probably lean to the simple map...

I will give it a try and update it shortly,.

Reply:
> > I think an explicit host field for indicating the interface to bind on is fine, even if that is likely to be unused on kubernetes. The thing I don't like is making it a sort of multi-purpose field.
> 
> Would it be better to delegate this to the options field then?

Like I say, I am fine with it being a field in its won right. I am also fine if it is an option. What I would rather avoid is using the same field/option for quite different things i.e. the interface to bind on or the host to assume for certain ingress types. I think those should be two distinct things and the latter should probably be an option since it will likely be specific to different types of ingress.

Reply:
I have removed the `host` parameter for now and just added the options (following the LinkAccess approach)
to Site, Connector and Listener specs.